### ISSUE TITLE
Kill an enemy NPC, LUA error

### ISSUE THREAD

**Issue by justinnichols** [Benik/BenikUI#43]
**Describe the bug and post the error message, if there is one**
After I kill an enemy NPC, I receive the LUA error listed below.  The main LUA error message is: `...dOns/ElvUI_BenikUI/modules/unitframes/unitframes.lua:166: attempt to index field 'BadDispels' (a nil value)`

**To Reproduce**
1. Log in with any character
2. Defeat an NPC Enemy

**Expected behavior**
No LUA error to occur.

**Please include the addon and game versions:**
 - BenikUI: v4.37
 - ElvUI: 13.33
 - WoW game variation (retail, WotLK, Classic Era): Retail 10.1

**Enabled Addons**
 - ElvUI
 - ElvUI BenikUI
 - ElvUI Libraries
 - ElvUI Options

**Additional context**
```
Message: ...dOns/ElvUI_BenikUI/modules/unitframes/unitframes.lua:166: attempt to index field 'BadDispels' (a nil value)
Time: Wed May 24 19:24:36 2023
Count: 874
Stack: ...dOns/ElvUI_BenikUI/modules/unitframes/unitframes.lua:166: attempt to index field 'BadDispels' (a nil value)
[string "=[tsm error check]"]:1: in main chunk
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_BenikUI/modules/unitframes/unitframes.lua"]:166: in function `PostUpdateButton'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua"]:301: in function <...e/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua:170>
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua"]:353: in function <...e/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua:345>
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua"]:516: in function `func'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua"]:1039: in function `run'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua"]:1069: in function `execute'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua"]:1089: in function `update'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua"]:1120: in function <Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua:1118>

Locals: (*temporary) = defined =[C]:-1
```

**Comment by Ommos** [comment]
Message: ...dOns/ElvUI_BenikUI/modules/unitframes/unitframes.lua:166: attempt to index field 'BadDispels' (a nil value)
Time: Wed May 24 23:46:50 2023
Count: 851
Stack: ...dOns/ElvUI_BenikUI/modules/unitframes/unitframes.lua:166: attempt to index field 'BadDispels' (a nil value)
[string "=[C]"]: ?
[string "@Interface/AddOns/ElvUI_BenikUI/modules/unitframes/unitframes.lua"]:166: in function `PostUpdateButton'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua"]:301: in function <...e/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua:170>
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua"]:353: in function <...e/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua:345>
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/elements/auras.lua"]:516: in function `func'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua"]:1039: in function `run'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua"]:1069: in function `execute'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua"]:1089: in function `update'
[string "@Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua"]:1120: in function <Interface/AddOns/ElvUI_Libraries/Core/oUF/ouf.lua:1118>

Locals: (*temporary) = <function> defined =[C]:-1

This is the error I get after killing an NPC

**Comment by Benik** [comment]
It's fixed on the dev version. I will push a new version as soon as l come back from work

**Comment by Benik** [comment]
Fixed on v4.38 for DF and v2.14 for Wrath/Classic

**Comment by justinnichols** [comment]
Thanks so much, @Benik !
